src/dmd/escape.d

```diff
@@ -1580,3 +1597,51 @@ else
     }
 }
 
+
+/**********************************************
+ * Have some variables that are maybescopes that were
+ * assigned values from other maybescope variables.
+ * Now that semantic analysis of the function is
+ * complete, we can finalize this by turning off
+ * maybescope for array elements that cannot be scope.
+ *
+ *  va      v      =>  va     v
```

Comment:
Are `va` and `v` referring to the foreach variables in the source below? If that’s the case please quote them with backticks to make that more clear.

Reply:
ok